backend/backend/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['0.0.0.0']
+ALLOWED_HOSTS = ['0.0.0.0','localhost']
```

Comment:
@rabbilyasar you mean here ?

Reply:
Yes between the host addresses add a space after the comma, that's a convention while writing lists. We will be following as much conventions as possible.